### ISSUE TITLE
Command "TSLint.showOutputChannel" not found with Problems from problem matcher not clearing

### ISSUE THREAD

**Issue by zikhan** [Microsoft/vscode-tslint#328]
- tslint Version: 5.9.1
- typescript Version: 2.6.2
-vscode-tslint Version: 1.0.28
-vscode version: 1.20.1

Steps to Reproduce:

1. Run `TSLint: Show Output` in vscode command palette.
2. Get message:
![image](https://user-images.githubusercontent.com/8488326/36493032-35b93956-16f4-11e8-8b0f-d2380bae7cd5.png)

Troubleshooting steps:
Uninstall/Reinstalled extension. Disabled/Re-enabled extension.
Set "tslint.trace.server" to both messages and verbose.

I do get TS Lint warnings on open files, however, I've attached the npm lint to the $tslint5 problem matcher and the problems will stick even after being modified and rerunning the task. The only way to clear the problems is restart vscode.

**Comment by egamma** [comment]
@zikhan I think you are reporting 2 problems.
1. the command is not found. We have seen this happening when the tslint extension was corrupt and this looks like a duplicate of #324
2. there is an issue with the $tslint problem matcher. Can you provide more details about the task you have defined. Can you share the task definition and attach it to the issue.

**Comment by zikhan** [comment]
1. Yep, that's the same problem. I uninstalled it, but shutdown vscode completely. I didn't actually close vscode after uninstalling the first time.
2.  The task is:
```JSON
{
    "type": "npm",
    "script": "lint",
    "problemMatcher": [
        "$tslint5"
    ]
}
```

_After Fixing the first issue_: I tried the task again and it worked. It cleared the problems reported by the task (The task definition above), and started reporting the same problems from the tslint.json file.
